File: 283-move-zeroes/283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        x=nums.count(0)
        for i in range(x):
            nums.remove(0)
            nums.append(0)
                
        

File: 283-move-zeroes/283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        vector<int>x;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]!=0)
                x.push_back(nums[i]);
        }
        for(int i=0;i<nums.size();i++)
        {
            if(i<x.size())
            {
                nums[i]=x[i];
            }
            else
            {
                nums[i]=0;
            }
        }
    }
};